README.md

```diff
@@ -387,6 +387,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Eve Online](https://esi.evetech.net/ui) | Third-Party Developer Documentation | `OAuth` | Yes | Unknown |
 | [Final Fantasy XIV](https://xivapi.com/) | Final Fantasy XIV Game data API | No | Yes | Yes |
 | [Fortnite](https://fortnitetracker.com/site-api) | Fortnite Stats | `apiKey` | Yes | Unknown |
+| [GamerPower](https://www.gamerpower.com/api-read) | Game Giveaways Tracker | No | Yes | Unknown |
```

Comment:
From the documentation looks the API has CORS. 
It'll be good to change `Unknown` to `Yes`.